sample-notificator-app/src/main/resources/notification-template/email-update-template.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>email-update-template</title>
+</head>
+<body>
+<div id="email-body">
+  <div id="person">Dear Max Mustermann,</div>
+  <br>
+  <div id="content">this is an automated email to inform you about recent updates on your subscribed project
+    <div id="project-information">My awesome project (ABCDE):</div>
+    <br>
+    <div id="sample-status-enumeration">
+      <div id="sample-status-failed-qc"><strong>X samples failed QC</strong>
+        <span id ="failed-qc-contact">(get in contact: <a href="mailto:support@qbic.zendesk.com">support@qbic.zendesk.com</a>)</span>
+      </div>
+      <div id="sample-status-available-data">X samples data available</div>
```

Comment:
How is the `X` intended to be filled? As it is not an element in the DOM it cannot be selected specifically this way. You could encapsulate it with a span and assign an id.

Reply:
The `X samples data available` should be `X samples have data available` I think

Reply:
That's a good point, however there should still be an outer div since this line should be deleted from the mail when there is no change in sample failed qc status

Reply:
Yes the outer div is fine 😄